data/baibhav-tiwari.json

```
@@ -3,8 +3,26 @@
   "displayStatsPublic": true,
   "type": "personal",
   "bio": "**Web3** | **Web-developer** | **React** | **Open_Source**",
```

Remove the underline it's not necessary
```suggestion
  "bio": "**Web3** | **Web-developer** | **React** | **Open Source**",
```